src/lib/FormGroupLayout.tsx

```diff
@@ -24,7 +24,16 @@ const FormGroupLayout = <T extends FieldValues>(props: FormGroupLayoutProps<T>)
   const switchLayout = layout === "switch";
   const checkboxLayout = layout === "checkbox";
 
-  return (
+  if (inputOnly && (switchLayout || checkboxLayout)) {
+    throw "'inputOnly' is not possible with switches or checkboxes";
```

Comment:
very nice 👏

Reply:
i would say it doesn't really makes sense to have that right? at least from our design its not that easy possible...

Reply:
yes, perfect like this :-)